code/modules/clothing/head/misc_special.dm

```diff
@@ -400,3 +400,35 @@
 	desc = "You feel ashamed about what you had to do to get this hat"
 	icon_state = "cowboy"
 	item_state = "cowboy"
+
+/obj/item/clothing/head/helmet/monkeytranslator
+	name = "\improper Uplift-O-Matic V1"
+	desc = "A complex array of machinery designed to boost the intelligence of monkeys. Nanotrasen is not responsible for misuse of this device. "
+	icon_state = "monkeyhat"
+	item_state = "monkeyhat"
```

Comment:
I hope you plan for these to be constructable in some way because in case of destruction or it being stolen the player is left screwed over.